liberapay/utils/__init__.py

```diff
@@ -109,6 +109,8 @@ def get_community(state, restrict=False):
 
 def b64decode_s(s, **kw):
     udecode = lambda a: a.decode('utf8')
+    if not isinstance(s, bytes):
+        s = s.encode('utf8')
```

Comment:
Oh is it? What is actual reason for 11b52fd30273cc213a9d4a5e8a86ee55615b386a to fail then?